woocommerce/woocommerce-ios
# [Woo POS] Design updates to item list and cart headers

Closes: #13686

## Description
This PR addresses the design tweaks found during walkthrough regarding the Item list and Cart view headers:
* Products and Cart headings should be aligned.
* The (i) icon should be aligned to the right, along the list of items.
* Additional space between `<` backbutton and Cart header is needed

## Screenshots
* Before: See [here](https://github.com/woocommerce/woocommerce-ios/issues/13686)
* After:

| After |
|--------|
| ![__ 20240822 aligned headers](https://github.com/user-attachments/assets/53b3ebd0-660b-4a52-8c8f-338b73fcbc5a) |
| ![__ 20240822 aligned headers 2](https://github.com/user-attachments/assets/677f9e36-eeb6-4d79-8eca-748106e671c4) | 

## Testing information
* Run POS
* Observe that the headers in the item list are aligned (Products, and (i) icon)
* Add some products to the cart
* Observe that the headers in the cart view are aligned (Cart, number of items, "Clear" button)
* Tap Checkout
* Observe that the headers in the cart view are aligned (backbutton, Cart header, number of items)

I tested both in simulator and physical devices, as well as by enabling the one-time simple product banner (you can test this by adding `UserDefaults.standard.set(false, forKey: "isSimpleProductsOnlyBannerDismissed")` to the ItemListViewModel init) and observed no regressions.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
